cluster/delegate.go

```diff
@@ -135,6 +135,7 @@ func (d *delegate) NotifyMsg(b []byte) {
 		level.Warn(d.logger).Log("msg", "decode broadcast", "err", err)
 		return
 	}
+	level.Debug(d.logger).Log("received", "NotifyMsg", "len", len(b), "key", p.Key)
```

Comment:
I'm not sure the extra logging is required.

Reply:
Done